src/WebJobs.Extensions.DurableTask/OutOfProcMiddleware.cs

```diff
@@ -639,15 +639,13 @@ private static FailureDetails GetFailureDetails(Exception e, out bool fromSerial
                         return details;
                     }
 
-                    // Try to extract properties from the serialized exception JSON
-                    IDictionary<string, object?>? properties = ExtractPropertiesFromExceptionJson(exception);
-
+                    // For non-.NET language, properties at FailureDetails is not supported yet.
                     if (TrySplitExceptionTypeFromMessage(exception, out string? exceptionType, out string? exceptionMessage))
                     {
-                        return new FailureDetails(exceptionType, exceptionMessage, stackTrace, innerFailure: null, isNonRetriable: false, properties: properties);
+                        return new FailureDetails(exceptionType, exceptionMessage, stackTrace, innerFailure: null, isNonRetriable: false, properties: null);
```

Comment:
What are the implications for Java with passing null to FailureDetails constructor here? Will we need to change this once Java SDK adds support for exception properties?

Reply:
Yeah it needs to be changed once java sdk support properties.

Reply:
yeah once we support Properties at FailureDetails, we need to go back here to update these as currently it will always be null.